### DISCUSSION TITLE
Error: Command "pnpm run build" exited with SIGABRT

### DISCUSSION THREAD

**Discussion by ThamizhiniyanCS** [vercel/community#5157] [Help]
### Summary

I am not able to deploy the nextjs project. I am successfully able to build the same project with same configuration with same repository on my local machine, as well I am able to successfully deploy it to netlify. But on Vercel, my project is not building and always throw me the error "Error: Command "pnpm run build" exited with SIGABRT", I tried deleting my project on vercel, created new project and again deployed it but still fails. 20 to 30 of my last build attempts were failed.

Screenshot of the error that is thrown by vercel during build:
![image](https://github.com/vercel/community/assets/95332789/f2aba706-80f0-4e17-b816-a9f96a4a60b0)

### Steps to Reproduce

The Error is with the build:
![image](https://github.com/vercel/community/assets/95332789/3e137aee-6ea5-4166-a250-f85cfe4f75ad)

**Comment by Linn-Htet123** [comment]
Have you solved it. I have faced the error like this too

**Comment by versecafe** [comment]
@Linn-Htet123 @Thamizhiniyan18 This has already been solved check out here https://github.com/orgs/vercel/discussions/5048 It's likely related to `sharp` versions, as SIGABRT is a c++ error message for malloc or any other call of `abort()`
